production-planning/src/test/scala/dev.atedeg.mdm.productionplanning/Tests.scala

```diff
@@ -0,0 +1,98 @@
+package dev.atedeg.mdm.productionplanning
+
+import java.util.UUID
+
+import cats.data.{ NonEmptyList, Writer }
+import cats.syntax.all.*
+import org.scalatest.GivenWhenThen
+import org.scalatest.featurespec.AnyFeatureSpec
+import org.scalatest.matchers.should.Matchers
+
+import dev.atedeg.mdm.productionplanning.*
+import dev.atedeg.mdm.productionplanning.OutgoingEvent.{ OrderDelayed, ProductionPlanReady }
+import dev.atedeg.mdm.products.{ CheeseType, Product }
+import dev.atedeg.mdm.utils.*
+import dev.atedeg.mdm.utils.given
+import dev.atedeg.mdm.utils.monads.*
+
+trait Mocks {
+
+  val cheeseTypeRipeningDays: CheeseTypeRipeningDays = Map(
+    CheeseType.Squacquerone -> RipeningDays(4),
+    CheeseType.Ricotta -> RipeningDays(0),
+    CheeseType.Caciotta -> RipeningDays(8),
+    CheeseType.Casatella -> RipeningDays(4),
+    CheeseType.Stracchino -> RipeningDays(5),
+  )
+
+  val prodToProd1: ProductToProduce = ProductToProduce(Product.Caciotta(500), Quantity(5))
+  val prodToProd2: ProductToProduce = ProductToProduce(Product.Casatella(300), Quantity(10))
+  val prodToProd3: ProductToProduce = ProductToProduce(Product.Squacquerone(250), Quantity(10))
+
+  val orderedProd1: OrderedProduct = OrderedProduct(Product.Caciotta(500), Quantity(5))
+  val orderedProd2: OrderedProduct = OrderedProduct(Product.Casatella(300), Quantity(10))
+  val orderedProd3: OrderedProduct = OrderedProduct(Product.Squacquerone(250), Quantity(10))
+}
+
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
+class Tests extends AnyFeatureSpec with GivenWhenThen with Matchers with Mocks {
+
+  Feature("Create the production plan for the day") {
+    Scenario("Raffaella wants to create the production plan") {
+      Given("a list of orders with deadline in 10 days")
+      val requiredBy = java.time.LocalDate.now.plusDays(10)
+      val orderedProducts = NonEmptyList.of(orderedProd1, orderedProd2, orderedProd3)
+      val orders = List(
+        Order(OrderID(UUID.randomUUID()), requiredBy, orderedProducts),
+        Order(OrderID(UUID.randomUUID()), requiredBy, orderedProducts),
+        Order(OrderID(UUID.randomUUID()), requiredBy, orderedProducts),
+      )
```

Comment:
You can use [`List.fill`](https://dotty.epfl.ch/api/scala/collection/immutable/List$.html#fill-3a1)
```suggestion
      val orders = List.fill(3)(Order(OrderID(UUID.randomUUID), requiredBy, orderedProducts)
```